frontend/src/text_resources.js

```diff
@@ -216,7 +219,7 @@ let LOCALIZE = new LocalizedStrings({
     emibTest: {
       //Home Page
       homePage: {
-        testTitle: "eMIB - Test Pratique",
+        testTitle: "FR Le eMIB",
```

Comment:
Do you want me to translate this one and commit in this PR?

Reply:
If you don't mind, please!

Reply:
Done in commit #https://github.com/code-for-canada/project-thundercat/pull/84/commits/f69b1e8aca6b3426cc25b4cf2812d7bcc3f9a1ab